### DISCUSSION TITLE
Is Solid's online/social media profile too inconsistent?

### DISCUSSION THREAD

**Discussion by bobaekang** [solidjs/solid#435] [General]
Although I've barely dipped my toe into using Solid, I've been a follower of Ryan's work for over a year now and wish for Solid's success.

With that said, I wonder if Solid's online/social media profile is a bit too inconsistent/fragmented:

* GitHub Org name is `solidui`
* Official twitter handle is `solid_js` (and name is `SolidJS`)
* Discord server vanity URL is `solidjs`
* Official website domain is `solidjs.com`

I remember that Ryan originally wanted to have a GitHub Org name to be something along the line with SolidJS, which was not available. In that sense, SolidUI sounds like a decent alternative.

But then, given the project's target audience (developers), I wonder if the GitHub Org name should be regarded something canonical/representative. If so, maybe it's worth consolidating the online/social media profiles around that before v1.0. Alternatively, maybe it helps to further differentiate the org from the flagship project (i.e. Solid the JS library) to avoid confusion (cf. Vercel-Next.js)?

The whole thing might not be a big deal after all. But I just wanted to bring this up just in case since Solid is getting close to its big day. 🚀

**Comment by ryansolid** [comment]
Yeah I mean a certain amount of this is hard at this point Vite.. couldn't get `@vite` or `@vitejs` on twitter either and settled for `@vite_js`. We had no problem except with Github and NPM. We actually could use `@solid_js` on npm. NPM I missed out on `@solidjs` by a few weeks, if I had registered it when I chose the name it would have been fine but I was a few months in before I thought of it.

I was really hoping for the github org since from the outside it looks like it is dead and has for years. But github assures me it is in use so I need to try to see if they'd be willing to give up the name.

The Vercel angle could work but I'm not really here trying to build out a company like that. I'd be tempted to just take a name with solid in it.. like we have. But that's where the disambiguity is. It almost needs to be completely different.

**Reply by bobaekang** [reply]
Thanks for your answer. Really wish it had worked out for you to take the `solidjs` name for the org. The current org 100% looks dead with its website (https://solidjs.io/) leading to nowhere. 🤷‍♂️ I guess that's life.

**Reply by luisherranz** [reply]
I guess you have probably thought about this already, but why not use `solidjsx`? I know Solid is not only about JSX, but it's close, and the `jsx` termination should be free pretty much everywhere (https://twitter.com/solidjsx, https://github.com/solidjsx, solidjsx.com...).

**Reply by ryansolid** [reply]
I have some good news. Look at the github org.

**Reply by bobaekang** [reply]
@ryansolid Congratulations!

**Reply by luisherranz** [reply]
Awesome 😎
